content/10.diagnostics.md

```diff
@@ -331,18 +331,56 @@ Further research is required to determine the full extent to which sustained imm
+The sensitivity and selectivities reported vary among CLIA tests and for the detection of IgM versus IgG, but sensitivities and selectivities as high as 100% have been reported among various high-throughput tests [@doi:10.1002/jmv.25932; @doi:10.1128/JCM.01361-20; @doi:10.1016/j.jcv.2021.104914].
+CLIA has previously been used to develop tests that can be used at point of care (e.g., [@doi:10.2147/IJN.S200556]) which may allow for this technique to become more widely accessible in the future.
+
+#### Lateral Flow Immunoassay
+
+The first serological test approved for emergency use in the United States was developed by Cellex [@url:https://www.fda.gov/media/136625/download].
+The Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay, also known as a lateral flow immunoassay, designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma (from a blood sample) of patients suspected to have developed a SARS-CoV-2 infection [@url:https://www.fda.gov/media/136625/download].
 The Cellex test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
 In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
 With this particular assay, results can be read within 15 to 20 minutes [@url:https://www.fda.gov/media/136625/download].
-Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai, proposed an ELISA test that detects IgG and IgM that react against the RBD of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
-The authors are now working to get the assay into clinical use [@url:https://www.livescience.com/coronavirus-tests-available.html].
+LFIs are often available at point of care but can have very low sensitivity [@doi:10.1128/JCM.01361-20].
+
+#### Neutralizing Antibody Assays
+
+Neutralizing antibody assays play a functional role in understanding immunity that distinguishes them from other serological tests.
+The tests described above are all binding antibody tests.
+On the other hand, rather than simply binding an antibody to facilitate detection, neutralizing antibody determines whether an antibody response is present that would prevent infection [@doi:10.1128/Spectrum.01202-21; @doi:10.1038/s41467-020-17892-0].
+Therefore, these tests serve the purpose of evaluating the extent to which a sample donor has acquired immunity that will reduce susceptibility to SARS-CoV-2.
+As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection and to assess vaccine candidates.
```

Comment:
```suggestion
As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection, to assess vaccine candidates, and to establish correlates of protection against infection and disease [@doi:10.1038/s41591-021-01377-8; @doi:10.1128/JCM.02107-20; @doi:gnx76z]
```

Reply:
In case I did the last one wrong: https://doi.org/10.1016/S2666-5247(21)00267-6

Reply:
@RLordan Close enough that I was able to figure it out! It's the string of letters/numbers with a 10/ in front!